### ISSUE TITLE
Calculate block rewards

### ISSUE THREAD

**Issue by elaineo** [ethereumproject/explorer#28]
https://github.com/ethereum/wiki/wiki/Mining

**Comment by realcodywburns** [comment]
Do you mean calculate an estimated return based on a users hash power based on difficulty?

**Comment by elaineo** [comment]
@realcodywburns no, the block reward + the gas that was paid via transactions. It lets people see how much ether miners are making per block.

**Comment by kimisan** [comment]
Wow cool, hope will have api for get this data.

**Comment by realcodywburns** [comment]
I was working the same problem from the other end. I was trying to estimate the base reward for a given hash rate.

The  successful miner of the winning block receives:
- A static block reward for the 'winning' block, consisting of exactly 5.0 Ether
- ALL of the gas expended within the block, that is, all the gas from the contracts that were run within the block submitted by the winning miner. Over time, it's expected these will dwarf the static block reward
- An extra reward for including Uncles as part of the block, in the form of an extra 1/32 per Uncle included

Uncles included in a block formed by the successful PoW miner receive:
- 7/8 of the static block reward

My work porting a simple mining calculator: (need an ETC api)  https://github.com/realcodywburns/ethereum-mining-calculator
